utils/__snapshots__/message_html_to_component.test.jsx.snap

```diff
@@ -45,10 +45,12 @@ Array [
   "
```

Comment:
Changes in this file probably related to not needed reformating.

Reply:
I did not change anything on `message_html_to_component`, I've just pulled the upstream master into this branch. And ran, `npm test`